hera_mc/mc_session.py

```diff
@@ -1786,17 +1735,14 @@ def node_power_command(self, nodeID, part, command, nodeServerAddress=None,
             If the specied node is not in the array and `testing` is False.
 
         """
-        from .node import (NodePowerCommand, power_command_part_dict,
-                           get_node_list, defaultServerAddress)
-
         if nodeServerAddress is None:
-            nodeServerAddress = defaultServerAddress
+            nodeServerAddress = node.defaultServerAddress
 
         if testing:
             node_list = list(range(1, 31))
             dryrun = True
         else:
-            node_list = get_node_list(nodeServerAddress=nodeServerAddress)
+            node_list = node.get_node_list(nodeServerAddress=nodeServerAddress)
```

Comment:
It looks like this isn't covered by tests right now. If we're expecting this (because this is "live-only" functionality), then we should add a `# pragma: no cover` here.

Reply:
This is now covered by a test on the CI, thanks for catching it!